lms/migrations/versions/2f05ff66ec2e_courseroster_table.py

```diff
@@ -0,0 +1,54 @@
+"""CourseRoster table."""
+
+import sqlalchemy as sa
+from alembic import op
+
+revision = "2f05ff66ec2e"
+down_revision = "b97080de93bc"
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "course_roster",
```

Comment:
What is every row in this table supposed to represent?

Reply:
Wait, I think I actually get it. Is it one row for every user who's part of a course, with the rule of that user in that course?

Reply:
Yes sorry, this and the model PR are bit sparse in the details. I think all this make more sense starting from:

https://github.com/hypothesis/lms/pull/6609

I'll add a few comments to the model code anyway :+1: